cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1937,6 +1983,11 @@ search = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" _index_earliest=-
 | sort - count \
 | `cs_email_daily_spam_email_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" Status=FilteredAsSpam 
+action.cyences_notable_event_action.system_compromised_search = | stats count by SenderAddress
```

Comment:
The Attacker and Compromised system need to be reversed. (Debatable, we'll discuss on call further.)

Reply:
As discussed in call. This is fine.